lib/utils/check.ts

```diff
@@ -0,0 +1,64 @@
+import isEmpty from 'lodash/isEmpty';
+import isPlainObject from 'lodash/isPlainObject';
+import type { DataType } from '../..';
+import { getOperators } from './format';
+// eslint-disable-next-line import/order
+import { Where } from './sequelize-method';
+
+// eslint-disable-next-line @typescript-eslint/no-require-imports
```

Comment:
I assume this will be removed after we have migrated data-types to TS?

Reply:
Will not duplicate this comment over the other files since I think the reason will be the same in each case

Reply:
Might add a TODO before this line to mention this? Especially considering my single comment from before about unicorn/expiring-todo-comments

Reply:
Later you mention "-- .js files must be imported using require" which solves my answer. Still think the TODO might be nice

Reply:
Will add the comment everywhere & add TODO :)

Reply:
Does the plugin still work though? Or do we maybe want to fork it? Was thinking of forking eslint-plugin-small-import since installing your config will give a warning about the peerDependency not matching. Maybe we can make a special plugin with rules unmaintained elsewhere that we use internally?

Reply:
`eslint-plugin-small-import` would be an easy fork too, we just need to update its peerDependencies declaration

https://github.com/mysticatea/eslint-plugin-eslint-comments is probably more work but also much more useful for sequelize